### ISSUE TITLE
help / not covered in the book

### ISSUE THREAD

**Issue by jpgpi250** [StamusNetworks/suricata-4-analysts#2]
not an issue, just an attempt to get help.

I'm trying everything I can to keep DoH / DoT / DoQ out of my network.

I've already published several methods to achieve this, my manual [here](https://jpgpi250.github.io/piholemanual/doc/Block%20DOH%20with%20pfsense.pdf), my suricata rules [here](https://raw.githubusercontent.com/jpgpi250/piholemanual/master/DOH/DOH.rules), daily updated.

Now looking into the data, returned by an SVCB DNS query.
initial discussion (pi-hole forum) [here](https://discourse.pi-hole.net/t/are-svcb-queries-sent-through-pi-hole-of-blocking-concern/59983/22)
question on the suricata forum [here](https://forum.suricata.io/t/suricata-rule-to-block-dns-query-type-64-svcb/3088)

TLDR; My opinion, SVCB data can be used by “rogue” applications to bypass the system configured DNS server, therefore, I would like to add a suricata rule that blocks SVCB (type 64) DNS queries.

**The question**, given wireshark is able to identify type 64 queries, how would suricata be able to block / reject these DNS queries

![image](https://user-images.githubusercontent.com/24669386/209451055-cb69943c-6637-41e3-b6d8-51b1035bc54e.png)

Thanks for your time and effort.
